demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -245,3 +248,22 @@ def test_is_all_params_not_hidden(self, source, answer):
         structure = StructureValidator(source)
         validator = IntegrationValidator(structure)
         assert validator.is_all_params_not_hidden() is answer
+
+    with open(GIT_HAVE_MODIFIED_AND_NEW_FILES, "r") as test_params_file:
+        tests_params = json.load(test_params_file)
+    params = [
+        (None, tuple(set(i) for i in tests_params['data']['params_with_data']), '123456', True),
+        ('origin/master', tuple(set(i) for i in tests_params['data']['params_with_data']), '123456', True),
+        (None, tuple(set(i) for i in tests_params['data']['params_with_data']), '', True),
+        (None, tuple(set(i) for i in tests_params['data']['params_without_data']), '123456', True),
```

Comment:
please add a test for `file_validator._is_valid is False`

Reply:
done